media/Circletracer/loadfile.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let jsonLoadCheck = 0;
+function openFileSelector(flag) {
+  const input = document.createElement('input');
+  input.type = 'file';
+  input.accept = 'text/plain';
```

Comment:
Circletracer should only read `json` files, but `text/plain` is not the correct type.

Reply:
I missed it. Thank u :)

Reply:
@llFreetimell 
Which is the extension of the file coming through the backend, txt or json?

Reply:
json is correct :)